### PR TITLE
<AddItem /> - added icon prop for illustrations

### PR DESCRIPTION
**Pull request by egidijusja**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6786/

**Comment by egidijusja** [comment]
Hey @milkyfruit, we have added `icon` to enable users to use illustrations by providing src of them. I have added one example for it. Can you review it? Also `icon` prop allows user to pass react node (e.g. he wants some custom icon/component instead of default addIcon) - should we have example for this case as well?

**Review by milkyfruit — CHANGES_REQUESTED** [review]
- [ ] Change prop name from `icon` to `illustration`. It will match the zeplin specs nad Modal API which has a similar case.
- [ ] Simplify the example by showing only the first option. The second option doesn't follow design rules.
```
<AddItem icon="generic_add_item_illustration.svg" size="large">
  Drag here to upload
</AddItem>
```

@egidijusja

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-ui-tpa**

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6786/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-style-react**

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6786

**Comment by egidijusja** [comment]
> * [ ]  Change prop name from `icon` to `illustration`. It will match the zeplin specs nad Modal API which has a similar case.
> * [ ]  Simplify the example by showing only the first option. The second option doesn't follow design rules.
> 
> **Overriding Icon**
> Component allows to override the icon with an illustration. It’s useful when add item contains important functionality that needs emphasis. For example drag and drop file uploading.
> 
> ```
> <AddItem icon="generic_add_item_illustration.svg" size="large">
>   Drag here to upload
> </AddItem>
> ```
> 
> I think it doesn't need additional example showing how a node works.
> 
> @egidijusja

As discussed on call will leave `icon` prop name to ensure consistency inside component with other prop `showIcon`. Other things were fixed.

**Review by milkyfruit — APPROVED** [review]

**Review by mykas — APPROVED** [review]

**Comment by mykas** [comment]
@jonathanadler Please review the changes for wix-ui-tpa.

**Comment by jonathanadler** [comment]
> @jonathanadler Please review the changes for wix-ui-tpa.

@sivanharel can you review?

File: packages/wix-ui-tpa/perfer.config.js [hunk 1]
```diff
@@ -4,9 +4,9 @@ const files = {
   'ActionsMenuLayout.bundle.min.js': 5.93,
   'ActionsMenuLayoutPerfBasic.bundle.min.js': 6.046,
   'ActionsMenuLayoutPerfExtended.bundle.min.js': 6.606,
-  'AddItem.bundle.min.js': 40.344,
-  'AddItemPerfBasic.bundle.min.js': 40.387,
-  'AddItemPerfExtended.bundle.min.js': 40.669,
+  'AddItem.bundle.min.js': 42,
+  'AddItemPerfBasic.bundle.min.js': 42,
+  'AddItemPerfExtended.bundle.min.js': 43,
```
Comment (sivanharel) [inline packages/wix-ui-tpa/perfer.config.js:9]:
```suggestion
  'AddItem.bundle.min.js': 41.701,
  'AddItemPerfBasic.bundle.min.js': 41.942,
  'AddItemPerfExtended.bundle.min.js': 42.475,
```

Comment (sivanharel) [reply packages/wix-ui-tpa/perfer.config.js:9]: We set the actual bundle size here in order to know what is the real size.
We do have a 1kb delta which is the max size that is allowed.

Screen Shot 2021-02-28 at 9 34 57

**Review by sivanharel — CHANGES_REQUESTED** [review]
Looks good!
Just change the bundle sizes according to my comment :)

**Review by sivanharel — APPROVED** [review]

**Pull request merged**